admin-src/DiagnoseResponse/index.js

```diff
@@ -74,20 +81,22 @@ export const DiagnoseResponse = ({ query }) => {
     clearScreenIfCurrentDiagnoseWasAnsweredByAnotherOne()
   }, [diagnoses])
 
-  const handleSubmit = async (values) => {
+  const handleSubmit = async values => {
     setIsSubmitting(true)
 
     const writeNewResponse = async () => {
       const newResponse = {
         answer: values.answer,
         answeredBy: values.answeredBy,
         answeredByUID: firebase.auth().currentUser.uid,
-        createdAt: firebase.firestore.FieldValue.serverTimestamp()
+        createdAt: firebase.firestore.FieldValue.serverTimestamp(),
       }
 
       await firebase
         .firestore()
-        .collection(`users/${currentDiagnose.userUID}/requests/${currentDiagnose.id}/responses`)
+        .collection(
+          `users/${currentDiagnose.userUID}/requests/${currentDiagnose.id}/responses`,
```

Comment:
Does it work ? 
I don't know if a comma at the end of parameters list does anything but seems wrong to me. I also think it would be better as a single line.

Reply:
> Does it work ?
> I don't know if a comma at the end of parameters list does anything but seems wrong to me. I also think it would be better as a single line.

Yes, it works. JavaScript supports trailing commas in a number of places, so that you don't have to add them when adding a new parameter/element to an array or object. That way, a diff of the changes for a file only show the line that adds the new element as changed and not the previous one where only a comma was added. [More information](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Trailing_commas).

Having said that, **we follow StandardJS and it doesn't allow trailing commas**.